pino.js

```diff
@@ -258,7 +259,7 @@ Pino.prototype.child = function child (bindings) {
   return new Pino(
     bindings.level || this.level,
     this.stream,
-    this.serializers,
+    extend(this.serializers, bindings.serializers),
```

Comment:
Lets aim for it!

On 26 Jul 2016 6:08 p.m., "Matteo Collina" <notifications@github.com> wrote:

> In pino.js <https://github.com/mcollina/pino/pull/75#discussion_r72293491>
> :
>
> > @@ -258,7 +259,7 @@ Pino.prototype.child = function child (bindings) {
> >    return new Pino(
> >      bindings.level || this.level,
> >      this.stream,
> > -    this.serializers,
> > +    extend(this.serializers, bindings.serializers),
>
> My goal is, there should be no perf degradation *if bindings.serializers
> is not passed in*. If there are by setting serializers, it's fine as it
> is an edge case anyway.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mcollina/pino/pull/75/files/446f62200f0db98b62c9b9bc5cb05656da29d9dd#r72293491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHkOiwpzU_NE4RIx5wl2EfOqn_oXPF3ks5qZj8YgaJpZM4JUo7K>
> .
>